### PR TITLE
Remove unused `is_k1?` from ApplicationHelper, clean up `Script#k1?`

### PR DESCRIPTION
**Pull request by joshlory**

### CODE REVIEW

File: dashboard/test/models/script_test.rb [hunk 1]
```diff
@@ -674,4 +674,12 @@ def populate_cache_and_disconnect_db
     assert_equal 'Stage 1 is pretty neat', updated_report_script['stages']['Report Stage 1']['description_student']
     assert_equal 'This is what you should know as a teacher', updated_report_script['stages']['Report Stage 1']['description_teacher']
   end
+
+  test 'text_to_speech_enabled? when script k1? is true' do
+    assert Script.find_by_name('course1').text_to_speech_enabled?
+  end
+
+  test '!text_to_speech_enabled? by default' do
```
Comment (ashercodeorg) [inline dashboard/test/models/script_test.rb:682]: nit: Not sure if we usually use negations (!) in test names like this.

**Comment by ashercodeorg** [comment]
A bunch of levels (e.g., [dashboard/config/scripts/levels/K-1 Maze Assessment 13.level](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts/levels/K-1%20Maze%20Assessment%2013.level#L13)) have `is_k1` defined. Is there cleanup there also possible?

**Comment by joshlory** [comment]
The `is_k1` (no question mark) property gets mapped to `isK1` property on `appOptions.level`, which is used by some apps to render K1 blocks: https://github.com/code-dot-org/code-dot-org/blob/723ad36e2c67254db3b89448dc31c7d5ee5f6b73/apps/src/maze/collectorBlocks.js#L7

**Comment by ashercodeorg** [comment]
Ah. Thanks.

**Pull request merged**